runtime/lua/vim/lsp/protocol.lua

```diff
@@ -635,6 +635,16 @@ export interface WorkspaceClientCapabilities {
 function protocol.make_client_capabilities()
   return {
     textDocument = {
+      inlayHint = {
+        dynamicRegistration = false,
+        resolveSupport = {
+          properties = {
+            'kind',
+            'textEdits',
+            'tooltip', --[[???]]
```

Comment:
`label.tooltip`, `label.command`, `label.location` would need to be added for completeness, if I'm reading the spec right.